include/ttmlir/Dialect/TTNN/IR/TTNNOps.td

```diff
@@ -63,6 +63,14 @@ class TTNN_ElementwiseOp<string mnemonic, list<Trait> traits = []> :
     let results = (outs Variadic<AnyRankedTensor>:$results);
 }
 
+class TTNN_ElementwiseUnaryOp<string mnemonic, list<Trait> traits = []> :
+    TTNN_ElementwiseOp<mnemonic, traits> {
```

Comment:
The only thing I don't like here is that inputs and outputs are variadic for a unary op. Should we extend from `TTNN_NamedDPSOp` instead?

Reply:
Didn't know that was an option, sounds great!

Reply:
Thanks for the suggestion, I will make a default builder for unary op definitely!

Reply:
Added custom builders for unary and binary element-wise ops!